.github/workflows/lib-injection.yml

```diff
@@ -3,77 +3,66 @@ on:
   push:
 
 jobs:
-  build-and-publish-init-image:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v3
-
-    - name: Set up QEMU
-      uses: docker/setup-qemu-action@v2
-
-    - name: Set up Docker Buildx
-      id: buildx
-      uses: docker/setup-buildx-action@v2
-
-    - name: Available platforms
-      run: echo ${{ steps.buildx.outputs.platforms }}
-
-    - name: Set Docker Image Tag
-      id: set_names
-      run: |
-        DOCKER_IMAGE_TAG=${{ github.sha }}
-        DOCKER_IMAGE_NAME=dd-lib-python-init
-        DOCKER_IMAGE_NAME=$(echo ghcr.io/${GITHUB_REPOSITORY}/${DOCKER_IMAGE_NAME} | tr '[:upper:]' '[:lower:]')
-        DOCKER_IMAGE_NAME_WITH_TAG=$(echo ${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG} | tr '[:upper:]' '[:lower:]')
-        echo "Using image name '$DOCKER_IMAGE_NAME_WITH_TAG'"
-        echo "::set-output name=image_name::$DOCKER_IMAGE_NAME_WITH_TAG"
-    - name: Login to Docker
-      run: docker login -u publisher -p ${{ secrets.GITHUB_TOKEN }} ghcr.io
-    - name: Docker Build
-      uses: docker/build-push-action@v3
-      with:
-        push: true
-        tags: ${{ steps.set_names.outputs.image_name }}
-        platforms: "linux/amd64,linux/arm64/v8"
-        context: ./lib-injection
-
-  build-and-publish-sample-app-images:
+   injection-tests-python:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
     strategy:
       matrix:
-        image-name: [
-          "dd-lib-python-init-test-django",
-          "dd-lib-python-init-test-django-gunicorn",
-          "dd-lib-python-init-test-django-uvicorn",
-        ]
+        lib-injection-connection: [ 'network','uds']
+        lib-injection-use-admission-controller: ['', 'use-admission-controller']
+        weblog-variant: [ 'dd-lib-python-init-test-django','dd-lib-python-init-test-django-gunicorn','dd-lib-python-init-test-django-uvicorn']
+      fail-fast: false
+    env:
+      TEST_LIBRARY: python
+      WEBLOG_VARIANT: ${{ matrix.weblog-variant }}
+      LIBRARY_INJECTION_CONNECTION: ${{ matrix.lib-injection-connection }}
+      LIBRARY_INJECTION_ADMISSION_CONTROLLER: ${{ matrix.lib-injection-use-admission-controller }}
+      DOCKER_REGISTRY_IMAGES_PATH: ghcr.io/datadog/system-tests
+      DOCKER_IMAGE_TAG: ${{ github.sha }}
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout python tracer
+        uses: actions/checkout@v3
+        with:
+          path: './dd-trace-py'
+ 
+      - name: Checkout system tests
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # 2.3.4
+        with:
+          repository: 'DataDog/system-tests'
+          ref: 'robertomonteromiguel/lib_injection_integration'
```

Comment:
I will remove this:

When my system-tests branch is merged to main branch
Just before merging this PR